src/main/java/hng_java_boilerplate/externalPages/services/AboutPageInitializer.java

```diff
@@ -0,0 +1,53 @@
+package hng_java_boilerplate.externalPages.services;
+
+import hng_java_boilerplate.externalPages.entity.AboutPage;
+import hng_java_boilerplate.externalPages.entity.CustomSection;
+import hng_java_boilerplate.externalPages.entity.ServiceEntity;
+import hng_java_boilerplate.externalPages.entity.StatEntity;
+import hng_java_boilerplate.externalPages.repository.AboutPageRepository;
+import hng_java_boilerplate.externalPages.repository.CustomSectionRepository;
+import hng_java_boilerplate.externalPages.repository.ServiceRepository;
+import hng_java_boilerplate.externalPages.repository.StatRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.context.event.ApplicationReadyEvent;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Service;
+@RequiredArgsConstructor
+@Service
+public class AboutPageInitializer {
```

Comment:
add space between the import and your code

Reply:
I have added space between them